services/frontend-v3/src/components/diplomaTable/DiplomaTable.jsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState, useEffect } from "react";
+import DiplomaTableView from "./DiplomaTableView";
+import { Skeleton } from "antd";
+import axios from "axios";
+import _ from "lodash";
+import { injectIntl } from "react-intl";
+import config from "../../config";
+
+const backendAddress = config.backendAddress;
+
+function DiplomaTable(props) {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [reset, setReset] = useState(false);
+  const [searchText, setSearchText] = useState("");
+  const [searchedColumn, setSearchedColumn] = useState("");
+  const [selectedRowKeys, setSelectedRowKeys] = useState([]);
+  const [size] = useState("large");
```

Comment:
if you are never going to set the sate then why use a state. you can just make variable. States should only be used for variables that will be updated (and thus would need a rerender)

Reply:
Fixed.